index.html

```diff
@@ -289,6 +289,28 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <footer class="contact_form">
+        <h2 class="contact_header">
+          I'm always interested in hearing
+           about new projects, 
+          so if you'd like to
+           chat please get in touch.
+        </h2>
+        <form action="#" method="post" class="form">
+          <label for="name" class="label_name">
+            <input type="text" name="Full name" id="name" placeholder="Full name" maxlength="30" />
+          </label>
+          <label for="email" class="label_email">
+            <input type="email" name="email" id="email" placeholder="Email address" />
+          </label>
+          <label for="text" class="label_message">
+            <textarea name="text" id="text" cols="30" rows="10" maxlength="500" placeholder="Enter text here" ></textarea>
+          </label>
+          <label for="submit" class="label_submit">
+            <button type="submit" id="submit">Get in touch</button>
+          </label>
+        </form>
```

Comment:
Currently, when I submit my contact without inputting any content It submits, It would be lovely 👍🏾  if you have checks to make sure every content in the form is filled 😄